pubsub.go

```diff
@@ -67,12 +67,24 @@ func (c *PubSub) _conn(channels []string) (*pool.Conn, error) {
 func (c *PubSub) resubscribe(cn *pool.Conn) error {
 	var firstErr error
 	if len(c.channels) > 0 {
-		if err := c._subscribe(cn, "subscribe", c.channels...); err != nil && firstErr == nil {
+		channels := make([]string, len(c.channels), 0)
```

Comment:
You need to remove `, 0` here and below to fix the build

Reply:
Done. Also once you merge this, can you create a new release, I need this fix to use this package in my new project.

Reply:
Released v6.10.2